### PR TITLE
Add translated files.

### PR DESCRIPTION
**Pull request by richardpringle**

### CODE REVIEW

**Comment by 0candy** [comment]
@superkhau @gunjpan PTAL

File: .gitignore [hunk 1]
```diff
@@ -23,5 +23,5 @@ checkstyle.xml
 example/server-2.0/client/public/vendor
 doc
 
-intl/*
-!intl/en/
+
+
```
Comment (superkhau) [inline .gitignore]: remote empty lines

File: intl/de/messages.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "db1e3eb9bf0934d72eb8b0b110281f39": "Ungültiges Feld: {0}",
+  "e254b20a16461c7379202f6a5f8350cc": "Ungültiger Variablentyp für ${{0}}",
+  "f5146dece5cd70db519daf8c7e6d8478": "${{0}} ist nicht vorhanden",
+  "f95c75ae3da1e88794fff4be7188f3dc": "Ungültiger Wert für `{0}`",
+  "e9023b2ab0a052c9ccbd257198c7429f": "{{APNS}}-Benachrichtigung kann nicht gesendet werden: {0}",
+  "704e7bf9910b8532f7c4889366fdcbac": "{{GCM}}-Fehlercode: {0}, deviceToken: {1}",
+  "033d31e7bd62a420f23e4b154945a2b8": "notification muss ein Objekt sein",
+  "1ec12f42dab8979b9fc90a95a023419e": "deviceTokens muss ein Array sein"
+}
+
```
Comment (superkhau) [inline intl/de/messages.json]: extra empty line

Comment (superkhau) [inline intl/de/messages.json]: in all the files

**Comment by 0candy** [comment]
Created issue for CI failure on windows.
https://github.com/strongloop/loopback-component-push/issues/127

**Pull request merged**